### DISCUSSION TITLE
React version ^18.0.0, MobX version ^6.6.1. Getting error ' [MobX] Proxy not available' on Android 7. Has anyone experienced this issue before?

### DISCUSSION THREAD

**Discussion by fruitGirl** [mobxjs/mobx#3688] [Q&A]
> 这个问题导致白屏，有没有大佬可以帮忙看下

Can any expert help me take a look at this issue that is causing a white screen? --translated by chatgpt

**Comment by fruitGirl** [comment]
<img src="https://user-images.githubusercontent.com/24424584/235044569-bea2890f-47ac-463b-ab72-d9cdd8050b73.jpeg" width="375px">

**Comment by iChenLei** [comment]
If you can't speak in English, please use ChatGPT to translate your question at least. Thanks. For your question, you can read this doc https://mobx.js.org/configuration.html#proxy-support , hope it can help you solve your issue.

**Reply by fruitGirl** [reply]
"Thank you for your help. I made the browser compatibility adaptation above, but it also reported other errors."
<img width="375" alt="image" src="https://user-images.githubusercontent.com/24424584/235063294-1ca9bad2-c50a-41f3-a517-eb51aefdd85e.jpeg">
My data is used as an object, but do I need to use a different ？Can you guide me, please?

**Reply by fruitGirl** [reply]
<img src="https://user-images.githubusercontent.com/24424584/235064277-9db97be2-1a89-4600-aebf-3c8dc55a5c61.jpeg" width="375px">

**Reply by fruitGirl** [reply]
<img width="742" alt="WechatIMG336" src="https://user-images.githubusercontent.com/24424584/235064433-41f9589c-880d-44aa-844e-b5e04059138e.png">

**Reply by iChenLei** [reply]
> read this doc https://mobx.js.org/observable-state.html#makeobservable
```ts
const store = makeAutoObservable({
  data: {
     localStorageInfo: ''
  },
  setLocalStorageInfo (val) {
   // use `this`, not `store` 
    this.data.localStorageInfo = val;
  }
})
```
